rules/concepts.md

```diff
@@ -22,4 +23,59 @@ formerly implicit variables
 How to find and install.
 How to write will be on a later page.
 
-### Comprehensive Examples
\ No newline at end of file
+### Comprehensive Examples
+-->
+
+# Rules Concepts
+
+Thus far we've connected openHAB to devices through Things, modeled the devices with Items, discussed persistence and how to build the display to control your home.
+But all of that amounts to home control, not automation.
+To create home automation we need to define rules.
+
```

Comment:
Unlike the Getting Started Tutorial where each page build up to the next one, each page in the reference section of the docs should stand on their own. Given that, perhaps an intro more along the lines of 

> Rules are a way to define behaviors in openHAB. A rule takes the high level form of "when X occurs, if Y then do Z" where "X" are events that **trigger** a rule to run, "Y" are the **conditions** that must be true for the rule to run when triggered, and "Z" are the **actions** to perform when the rule is triggered and allowed to run.

Of course reword and rework to match what you have below. I think it's OK to repeat high level core ideas like this multiple times across the page. If the readers taken nothing else away from this page, it's these three parts of a rule.